CRD-DTR/TIRF/R4/resources/Questionnaire-R4-TIRF.json

```diff
@@ -0,0 +1,661 @@
+{
+    "resourceType": "Questionnaire",
+    "id": "TIRFRemsPatientEnrollment",
+    "name": "TIRFRemsPatientEnrollment",
+    "title": "TIRF Rems Patient Enrollment",
+    "status": "draft",
+    "subjectType": [
+      "Patient"
+    ],
+    "date": "2020-05-20",
+    "publisher": "Da Vinci DTR",
+    "extension": [
+      {
+        "url": "http://hl7.org/fhir/StructureDefinition/cqf-library",
+        "valueCanonical": "http://hl7.org/fhir/us/davinci-dtr/Library/BasicPractitionerInfo-prepopulation"
+      }
+    ],
+    "item": [
+      {
+        "linkId": "1",
+        "type": "group",
+        "text": "Patient Information",
+        "item": [
+          {
+            "linkId": "1.1",
+            "extension": [
+              {
+                "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire",
+                "valueCanonical": "questionnaire/patient-info-base"
+              },
+              {
+                "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire-expand",
+                "valueBoolean": true
+              }
+            ],
+            "type": "display"
+          },
+          {
+            "linkId": "1.2",
+            "text": "Race",
+            "type": "open-choice",
+            "required": false,
+            "repeats": true,
```

Comment:
I set this field to repeats since the patient enrollment pdf form showed it as asking for the patient to select all that apply. If this isn't something we need to have then I can remove it.

Reply:
Okay if it's in the original form than that's fine